pom.xml

```diff
@@ -151,6 +151,54 @@ under the License.
                 <skip.rya.it>false</skip.rya.it>
             </properties>
         </profile>
+
+        <!-- Enable this profile to generate a Code Coverage report with jacoco "mvn ... -P coverage" -->
+        <profile>
+            <id>coverage</id>
```

Comment:
Why have a profile?  Why not just run it as default as part of the build?

Reply:
I view code coverage as value added, especially since we are not enforcing a minimum coverage policy right now.  There is a minor cost associated with running it and it is conceivable that you might not want to run it as part of the release build.